provider/cloudflare/cloudflare.go

```diff
@@ -126,12 +133,30 @@ type CloudFlareProvider struct {
 	proxiedByDefault  bool
 	DryRun            bool
 	DNSRecordsPerPage int
+	RegionKey		  string
 }
 
+// func (p *CloudflareProvider) CreateDomainRecord(zoneID, recordType, name string, content string, ttl int) error {
+//     rec := cloudflare.DNSRecord{
+//         Type:    recordType,
+//         Name:    name,
+//         Content: content,
+//         TTL:     ttl,
+//         Proxied: p.proxied,
+//         // add the region key if provided
+//         Meta: map[string]interface{}{
+//             "region": p.config.CloudflareRegion, // Add this line
+//         },
+//     }
+
+//     // existing code
+// }
```

Comment:
Why is this code commented out?

Reply:
Sorry, I meant to delete that function. Just forgot